nrf_wifi/fw_if/umac_if/src/default/fmac_api.c

```diff
@@ -396,38 +394,23 @@ enum nrf_wifi_status nrf_wifi_fmac_dev_init(struct nrf_wifi_fmac_dev_ctx *fmac_d
 	}
 
 	nrf_wifi_osal_mem_set(fmac_dev_ctx->fpriv->opriv,
-			      rf_params,
-			      0xFF,
-			      sizeof(rf_params));
-
-	if (rf_params_usr) {
-		ret = nrf_wifi_utils_hex_str_to_val(fmac_dev_ctx->fpriv->opriv,
-						    rf_params,
-						    sizeof(rf_params),
-						    rf_params_usr);
+		             (unsigned char *)&phy_rf_params,
```

Comment:
Is a cast necessary here (cast is not being done below in fmac_api_comon.c). Also this initialization itself might not be needed here since `nrf_wifi_fmac_rf_params_get` seems to be doing this initialization internally.

Reply:
Done.